services/web/server/src/simcore_service_webserver/garbage_collector_core.py

```diff
@@ -362,7 +366,7 @@ async def _remove_single_orphaned_service(
             except (UserNotFoundError, ValueError):
                 user_role = None
 
-            project_uuid = currently_opened_projects_node_ids[service_uuid]
+            project_uuid = interactive_service.get("project_id", "")
```

Comment:
I think assigning ``project_uuid=""`` is not a good idea.  

Pls check what is done next line: it inquires whether user_id has permission to write a project whose uuid "you do not know".  Pretty strange request, right? :-)
Besides the fact I do not find it very useful,  the worst is that it might introduce another bug since I do not know how that function reacts to blank ``project_uuid`` ...

Reply:
So before doing this I've checked what the function does. It serialises the project_uuid to a string and makes a query to the DB. So the query will not return a reply.

Reply:
OK, I can change it to a proper dict key fetch